docs/en-US/component/loading.md

```diff
@@ -83,23 +83,29 @@ If Element Plus is imported entirely, a globally method `$loading` will be regis
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------- | ------------- |
+| target               | the DOM node Loading needs to cover. Accepts a DOM object or a string. If it's a string, it will be passed to `document.querySelector` to get the corresponding DOM node | ^[string] / ^[HTMLElement] | document.body |
+| body                 | same as the `body` modifier of `v-loading`                                                                                                                               | ^[boolean]                 | false         |
+| fullscreen           | same as the `fullscreen` modifier of `v-loading`                                                                                                                         | ^[boolean]                 | true          |
+| lock                 | same as the `lock` modifier of `v-loading`                                                                                                                               | ^[boolean]                 | false         |
+| text                 | loading text that displays under the spinner                                                                                                                             | ^[string]                  | —             |
+| spinner              | class name of the custom spinner                                                                                                                                         | ^[string]                  | —             |
+| background           | background color of the mask                                                                                                                                             | ^[string]                  | —             |
+| customClass          | custom class name for loading                                                                                                                                            | ^[string]                  | —             |
+| svg                  | custom SVG element to override the default loading spinner                                                                                                               | ^[string]                  | —             |
+| svgViewBox           | sets the viewBox attribute for loading svg element                                                                                                                       | ^[string]                  | —             |
+| beforeClose ^(2.8.0) | Function executed before loading attempts to close. If this function returns false, the closing process will be aborted. Otherwise, the loading will close.              | ^[Function]`() => boolean` | —             |
```

Comment:
```suggestion
| svg-view-box           | sets the viewBox attribute for loading svg element                                                                                                                       | ^[string]                  | —             |
| before-close ^(2.8.0) | Function executed before loading attempts to close. If this function returns false, the closing process will be aborted. Otherwise, the loading will close.              | ^[Function]`() => boolean` | —             |
```

Reply:
@tolking Thx for the review. But it seems to be a configuration item of the service, not a property configuration item of the directive. I think camel case is correct, just like `customClass`. May I know your thoughts on this?

Reply:
You are right.

It seems that this PR has no breaking changes, perhaps we can merge it before 2.8.0

`2.8.0` -> `2.7.8`

Reply:
我不用改吧？我记得那个版本号会自动替换。

Reply:
我怎么记得没有这个功能

Reply:
> 我怎么记得没有这个功能

我之前提其他pr也是写的下一个小版本，结果他们又让我改成下一个大版本了说可以自动替换，#14925就是一个例子

Reply:
并没有自动替换的功能，版本标记都是手动添加的，方便标记某些功能是哪个版本之后才有的。

Reply:
> 并没有自动替换的功能，版本标记都是手动添加的，方便标记某些功能是哪个版本之后才有的。

那我需要把版本改成2.7.8对吗？我已经懵了，我之前一直以为是下一个大版本😂

Reply:
@btea @tolking done~